www/gatsby-node.js

```diff
@@ -231,9 +232,13 @@ exports.createPages = ({ graphql, actions }) => {
         return undefined
       })
 
+      const releasedBlogPosts = blogPosts.filter(post =>
+        _.get(post, `node.fields.released`)
+      )
```

Comment:
I think we should use same graphql filter as in `www/src/templates/template-blog-list.js` so filtering happens inside query. We would need to split our `allMarkdownRemark` queries earlier into 2 separate ones - one for blog posts and one for the rests. Otherwise the `next` / `previous` might be unreliable

Reply:
I preferred this approach so we don't have to duplicate these lines: https://github.com/gatsbyjs/gatsby/blob/f06f6c37698ccdd91b0f550fa30ab55f1d47cae9/www/gatsby-node.js#L251-L266

The `next` problem was addressed in 81a1e50.

If you would prefer to separate the posts in the query anyways, I'll do that of course.

Reply:
No, that's fine 👍 - it produces same output